subsys/bluetooth/controller/ll_sw/nordic/lll/lll_df_internal.h

```diff
@@ -57,7 +57,7 @@ static inline uint8_t lll_df_sync_cfg_is_modified(struct lll_df_sync *df_cfg)
 
 /* Enables CTE reception according to provided configuration */
 void lll_df_conf_cte_rx_enable(uint8_t slot_duration, uint8_t ant_num, const uint8_t *ant_ids,
-			       uint8_t chan_idx, bool cte_info_in_s1);
+			       uint8_t chan_idx, bool cte_info_in_s1, uint8_t phy);
```

Comment:
Would it be better to have the below order of parameters:
```
void lll_df_conf_cte_rx_enable( uint8_t phy, uint8_t chan_idx, uint8_t slot_duration,
				uint8_t ant_num, const uint8_t *ant_ids, bool cte_info_in_s1);
```

Then, calls would look like:
```
lll_df_conf_cte_rx_enable(lll->phy_rx, data_chan_use, df_rx_params->slot_durations,
			  df_rx_params->ant_sw_len, df_rx_params->ant_ids,
			  CTE_INFO_IN_S1_BYTE);
```

Reply:
Why this way? There is a clear block of params related with DF at first, then any other additional parameters.

Reply:
No they are not mixed:
- sampling related first: slots duration, ant_num, and_id,
- other params chan_idx, CTEInfo placement in PDU, phy.

Who says macros should be at the end and PHY at the beginning? 
I haven't seen any rule about that in coding guidelines. 
Or is it just your preference?

Reply:
For me:
- phy and data_chan_use  are general BLE radio parameters
- slot duration, ant_num and cte_info_in_s1 are direction finding related

This is how you defined the below (DF related parameter after context and PDU):
```
/* Enables CTE transmission according to provided configuration */
void lll_df_cte_tx_enable(struct lll_adv_sync *lll_sync, const struct pdu_adv *pdu,
			  uint32_t *cte_len_us);
```

> Or is it just your preference?

yes, my preference based on how you used macros to pass parameters earlier in this PR
```
radio_df_ctrl_set(cte_len, RADIO_DFECTRL1_TSWITCHSPACING_2us,
			  RADIO_DFECTRL1_TSAMPLESPACING_2us, PHY_LEGACY);
```

Reply:
```
radio_df_ctrl_set(cte_len, RADIO_DFECTRL1_TSWITCHSPACING_2us, RADIO_DFECTRL1_TSAMPLESPACING_2us, PHY_LEGACY);
```
This is just a coincidence.

In my humble opinion is does not make sense to give order of parameters based on possible use of macros on caller site. 

```
void lll_df_cte_tx_enable(struct lll_adv_sync *lll_sync, const struct pdu_adv *pdu, uint32_t *cte_len_us);
```
Here `cte_len_us` is also used to return and `lll_sync` is a context pointer, hence I preferred to have context first and `out` parameter last.

Take into account that this is not first time the function parameters list is changed and you haven't requested any changes in this regard. I do not want to change the order of parameters because I've added a new one.

Reply:
fair  enough.